src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectReferenceFileController.java

```diff
@@ -93,12 +94,7 @@ public String getProjectReferenceFilesPage(final Model model, final Principal pr
 			map.put("label", file.getLabel());
 			map.put("createdDate", file.getCreatedDate());
 			Path path = file.getFile();
-			try {
-				map.put("size", Files.size(path));
-			} catch (IOException e) {
-				logger.error("Cannot find the size of file " + file.getLabel());
-				map.put("size", messageSource.getMessage("projects.reference-file.not-found", new Object[] {}, locale));
-			}
+			map.put("size", iridaFileStorageUtility.getFileSize(path));
```

Comment:
Want to add this method onto the `ReferenceFile` class the same as you added it to `SequenceFile` so it can get rid of the `IridaFileStorageUtility` dependency?

Reply:
I have this added in a separate branch